python/README.md

```diff
@@ -0,0 +1,186 @@
+![Timefold Logo](https://raw.githubusercontent.com/TimefoldAI/timefold-solver/main/docs/src/modules/ROOT/images/shared/timefold-logo.png)
+
+# Timefold Solver for Python
```

Comment:
If the TOML stuff etc. needs to be in the root of the repo, why does the README need to be in a subfolder? And wouldn't it make sense to have one unified README for the entire repo? 

I doubt that people will click to the "python" subdir to see a Python-specific README; they'll stay at the top level, and onl\ see Java-specific stuff.

Reply:
This is for the `pip` package description.

Reply:
(which cannot be `.adoc`)

Reply:
Unfortunate. This means we probably need to have some duplication in the top-level README, for the sake of not totally scaring away Python people.